runtime/bootstrapper.ts

```diff
@@ -7,26 +7,31 @@ export class SkyAppBootstrapper {
   public static bootstrapConfig: SkyAppBootstrapConfig;
 
   public static processBootstrapConfig(): Promise<any> {
-    let promises: Promise<any>[] = [];
-
     if (SkyAppBootstrapper.bootstrapConfig) {
+      let authPromise: Promise<any>;
+
+      // BBHelp init
+      if (SkyAppBootstrapper.bootstrapConfig.help) {
```

Comment:
This needs to follow what omnibar is doing which is checking to see if auth is enabled and then waiting for the token to be retrieved to load help.  Otherwise you'll load the help widget only to get redirected to the signin page if the user is not already signed in.